build.gradle

```
@@ -96,12 +96,13 @@ githubRelease {
    body = layout.projectDirectory.file('release/changes.md').asFile.text.trim()
 }
 
-tasks.register('createReleaseTag', CreateGitTag) {
+def createReleaseTag = tasks.register('createReleaseTag', CreateGitTag) {
+    mustRunAfter('publishPlugins')
```

i think we should lookup the task lazily here: https://docs.gradle.org/current/userguide/task_configuration_avoidance.html#sec:task_configuration_avoidance_pitfalls
```suggestion
    mustRunAfter(tasks.named('publishPlugins'))
```

---

Reading https://docs.gradle.org/current/dsl/org.gradle.api.Task.html#N18B56, it looks like passing a String should be fine, wdyt ?

---

I see what you mean, passing a string should work. My mistake. 👍